### PR TITLE
Add Templates doc

### PR DESCRIPTION
**Pull request by deandre**
## Change description

> New docs for Templates feature.

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

> 

## Tasks

- [ ] Screen shots looks fuzzy?

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: docs/templates.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+<img
+  className="full-mobile"
+  src="/docs/uploads/email-layout.png"
+  style={{ width: '80%', height: 'intrinsic' }}
+/>
+
+## Limitations
+
+- Templates work across channels subject to limitations for that channel. For instance, Slack does not support titles.
```
Comment (smeijer) [inline docs/templates.mdx:44]:
```suggestion
- Templates work across channels subject to limitations for that channel. For instance, Slack does not support headings.
```

File: docs/templates.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### Content
+
+Content is the body of your notification - the message you want to pass on to users. Content will be the body of an email and below the notification title for in-app notifications.
+
+The content input supports [HTML](https://html-online.com/editor/) and [Markdown](https://www.markdownguide.org/basic-syntax/).
+
+### Email Layout
+
+Please visit the Categories tab in the dashboard, then select Email Layout. Here, copy/paste the HTML of your current email template. Use the merge tag `{{ template.content }}` as a placeholder for the content. The content will default to `{{ notification.content }}` for uncategorized notifications and categories without configured templates.
```
Comment (smeijer) [inline docs/templates.mdx:32]:
```suggestion
Please visit the Channels tab in the dashboard, then select Email Layout. Here, copy/paste the HTML of your current email template. Use the merge tag `{{ template.content }}` as a placeholder for the content. The content will default to `{{ notification.content }}` for uncategorized notifications and categories without configured templates.
```

File: docs/templates.mdx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  className="full-mobile"
+  src="/docs/uploads/templates.png"
+  style={{ width: '80%', height: 'intrinsic' }}
+/>
+
+Merge tags such as `{{ notification.content }}` are supported — a complete list will appear when you press `{`, after which you can continue typing or select one from the list. More advanced functionality such as [control flow](https://shopify.github.io/liquid/tags/control-flow/) and [default values](https://shopify.github.io/liquid/filters/default/) are also supported — consult the [Liquid reference](https://shopify.github.io/liquid/basics/introduction/) more information.
+
+### Notification title
+
+The title will be the subject of emails, and the first bolded line for in-app
```
[the remark this answers is not in the document]
Comment (deandre) [reply docs/templates.mdx]: I detailed overrides a bit more under "How to compose templates".

File: docs/templates.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+notifications.
+
+### Content
+
+Content is the body of your notification - the message you want to pass on to users. Content will be the body of an email and below the notification title for in-app notifications.
+
+The content input supports [HTML](https://html-online.com/editor/) and [Markdown](https://www.markdownguide.org/basic-syntax/).
```
[the remark this answers is not in the document]
Comment (deandre) [reply docs/templates.mdx]: I tightened up the in-line explanations and added channel-specific styling notes under "Limitations".

**Comment by deandre** [comment]
@unamashana can I please have your 👀  on this?

**Comment by unamashana** [comment]
image

Otherwise looks good.

**Comment by deandre** [comment]
> image
> 
> Otherwise looks good.

Thanks! I've restructured the docs, splitting the components out into two articles. Mind giving it one final pass and if all looks good, your approval on the PR?

**Review by unamashana — APPROVED** [review]

**Pull request merged**